src/main/java/com/group6/harmoniq/controllers/QuizController.java

```diff
@@ -41,7 +39,8 @@ public class QuizController {
     private int recognitionScore = 0;
 
     @GetMapping("/quizzes/AlbumCoverQuiz")
-    public String startQuiz(Model model) {
+    public String startQuiz(HttpSession session, Model model) {
+        userController.incrementUserQuizCount(session); // Call incrementUserQuizzesPlayed
```

Comment:
I think we should increment the count once user has successfully finished the quiz

Reply:
See commit: 00cd1e83cb8fc00e9fd90a3c89100d4c3e29b77b